src/views/GameView.vue

```diff
@@ -1,9 +1,10 @@
 <template>
   <div id="game-view-wrapper">
-    <!-- Unauthenticated/Must re-log in -->
+    <!-- Unauthenticated/Must re-log in/ Unavailable game -->
     <template v-if="$store.state.game.myPNum === null">
```

Comment:
Do we still need this v-if? it looks like it'd be unnecessary with the logic in the `<game-unavailable-view>` itself

Reply:
without it, I get a blank game background.

Reply:
So, I was able to get the game to load properly, but upon removing this v-if I'm getting a typeerror 
` kingCount(player) {
      return player.faceCards.reduce((kingCount, card) => kingCount + (card.rank === 13 ? 1 : 0), 0);
    }, `  TypeError: Cannot read properties of null (reading 'faceCards')

Reply:
I'm comfortable leaving it for now. We can revisit this if it proves valuable to do so